librz/core/cmd_descs/cmd_descs.h

```diff
@@ -728,7 +728,7 @@ RZ_IPI RzCmdStatus rz_open_binary_list_ascii_handler(RzCore *core, int argc, con
 RZ_IPI RzCmdStatus rz_open_binary_show_handler(RzCore *core, int argc, const char **argv);
 RZ_IPI RzCmdStatus rz_open_binary_add_handler(RzCore *core, int argc, const char **argv);
 RZ_IPI RzCmdStatus rz_open_binary_file_handler(RzCore *core, int argc, const char **argv);
-RZ_IPI RzCmdStatus rz_open_binary_rebase_handler(RzCore *core, int argc, const char **argv);
+RZ_IPI RzCmdStatus rz_open_binary_rebase_handler(RZ_BORROW RzCore *core, int argc, RZ_BORROW const char **argv);
```

Comment:
```suggestion
RZ_IPI RzCmdStatus rz_open_binary_rebase_handler(RzCore *core, int argc, RZ_BORROW const char **argv);
```

Reply:
this file is automatically generated (see the header of the file), don't touch it manually.